src/utils/material.js

```diff
@@ -118,6 +148,9 @@ module.exports.updateDistortionMap = function (longType, shader, data) {
     if (texture && COLOR_MAPS.has(slot)) {
       rendererSystem.applyColorCorrection(texture);
     }
+    if (texture) {
+      module.exports.setTextureProperties(texture, data);
```

Comment:
I think we can name the function above:

```module.exports.setTextureProperties = function setTextureProperties(texture, data) {```

and call it here:

```
 if (texture) { setTextureProperties(texture, data); }
```

A bit cleaner